app/templates/components/builds-item.hbs

```diff
@@ -46,7 +46,7 @@
           short="true"
           repo=this.build.repo
           eventType=this.build.eventType
-          maxLength=50
+          maxLength=72
```

Comment:
* See https://travis-ci.community/t/display-complete-commit-title-in-travis-builds/6481/2 for why if there is a limit, existing 50 is the right limit.

Reply:
@native-api The article mention best practises for developers. The article also mentions that the average length is 50 chars, so should half of the commit messaged be cropped?

Reply:
I may be reading a different article but for me, the "“50 / 72” rule" that it's dedicated to reads:

> 1. The first line of your commit message must be maximum 50 characters long. No more, and (ideally), no less.

And this "first line" is exactly what is printed in `row-message`.

Reply:
Yes, the article discusses these limits later in order to justify them. But the resulting rule it proposes is this.

Reply:
In any case, note the _"if there is a limit"_ part. In the general comment, I proposed to abolish the limit altogether.

Reply:
@native-api

> The first line of your commit message must be maximum 50 characters long. No more, and (ideally), no less.

This is an advice for developer, but as you can see in the histogram, 50 chars are not sufficient as it crops 50% of the commit messages.

For me this PR is a quick fix and further improvement can be done in another PR or if you want to extend this directly...

Reply:
Oh, so you are not actually following that rule. Okay then.